src/libraries/Common/src/System/Runtime/InteropServices/LibraryImportAttribute.cs

```diff
@@ -3,32 +3,61 @@
 
 #nullable enable
 
-//
-// Types in this file are used for generated p/invokes (docs/design/features/source-generator-pinvokes.md).
-//
 namespace System.Runtime.InteropServices
 {
     /// <summary>
-    /// Indicates that method will be generated at compile time and invoke into an unmanaged library entry point
+    /// Attribute used to indicate a Source Generator should create a function for marshaling
+    /// arguments instead of relying on the CLR to generate an IL Stub at runtime.
     /// </summary>
     [AttributeUsage(AttributeTargets.Method, AllowMultiple = false, Inherited = false)]
-#if LIBRARYIMPORT_GENERATOR_TEST
+#if NET7_0_OR_GREATER
```

Comment:
We usually move attributes like these under CoreLib and use `#if SYSTEM_PRIVATE_CORELIB` to make them public/internal when they are included for downlevel targets. For example, look at attributes under `src\libraries\System.Private.CoreLib\src\System\Runtime\Versioning` or `src\libraries\System.Private.CoreLib\src\System\Diagnostics\CodeAnalysis`